kerckhoff/packages/models.py

```diff
@@ -240,13 +247,41 @@ def create_version(
             return package_version
 
     def publish(self):
-        # TODO: this is only for demo purposes, the actual publish needs to be significantly more customizable
-        integrations = self.package_set.integration_set.all()
-        for integration in integrations:
-            if integration.auth_data.active:
-                integration.publish(self.latest_version)
-        self.state = self.PUBLISHED
-        self.save()
+        """
+        Publishes article data to WordPress using REST API
+            - Sends article data as a HTML string
+            - Authentication using WordPress Application Passwords
+        """
+
+        #Extracting files from package version
+        packages_latest_version = self.get_version(self.latest_version.id_num).packageitem_set.all()
+        img_urls = {}
+        for file in packages_latest_version:
+            if(file.file_name == "data.aml"):
+                json_data = file.data
+            if("jpg" in file.file_name or "jpeg" in file.file_name
+               or "png" in file.file_name):
```

Comment:
Makes sense. I'll use the python os package for that. I don't remember if I only allow jpg, jpeg and png because DB only uses those file types, or if I simply forgot to include more. Do you know if kerckhoff should be able to support other image extensions as well?

Reply:
One thing they would want to have in the future is gifs sometimes. However, there some changes we need to make to aws uploading code to support gifs so for now we can leave it out